mix.exs

```diff
@@ -21,7 +21,9 @@ defmodule Scholar.MixProject do
   defp deps do
     [
       {:ex_doc, "~> 0.27", only: :dev, runtime: false},
-      {:nx, "~> 0.2.0"}
+      {:nx, github: "elixir-nx/nx", sparse: "nx"},
+      {:scidata, "~> 0.1.9"},
+      {:explorer, "~> 0.2.0"}
```

Comment:
```suggestion
      {:scidata, "~> 0.1.9", only: [:test, :dev]},
      {:explorer, "~> 0.2.0", only: [:test, :dev]}
```

The code itself doesn't depend on them, does it?

Reply:
It would be nice if we could not depend on them at all. Perhaps the tests can use a subset of the data parsed manually?

Reply:
Yes, only in the testing module at this point